regenerate.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+from typing import List
+
+import argparse
+import os
+import subprocess
+
+
+def run(*args) -> str:
+    return subprocess.run(
+        args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding='utf-8')
+
+
+def GatherPosts(posts_dir: str) -> List[str]:
+    post_paths = run('find', posts_dir, '-name',
+                     '*.md').stdout.strip().split('\n')
+    return post_paths
+
+
+def GenerateHtml(pandoc_flags: str, post_paths: List[str]):
+    for post_path in post_paths:
+        output_path = post_path[:-2] + 'html'
+        run('pandoc', '-s', pandoc_flags
+            if pandoc_flags else '', post_path, '-o', output_path)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--posts',
+        type=str,
+        default='algos',
+        help='directory relative to the repo root')
+    parser.add_argument(
+        '--pandoc_flags', type=str, help='passed verbatim to pandoc')
```

Comment:
There's presumably a set of default flags we'll want to apply?

Reply:
The plan is to add that later. Right now, I'm not confident enough about what they'll be.